consensus/poet/families/sawtooth_validator_registry/validator_registry/processor/handler.py

```diff
@@ -45,6 +46,62 @@
     hashlib.sha256("validator_registry".encode()).hexdigest()[0:6]
 
 
+# Constants to be used when constructing config namespace addresses
+_CONFIG_NAMESPACE = '000000'
+_CONFIG_MAX_KEY_PARTS = 4
+_CONFIG_ADDRESS_PART_SIZE = 16
+
+
+def _config_short_hash(byte_str):
+    # Computes the SHA 256 hash and truncates to be the length
+    # of an address part (see _config_key_to_address for information on
+    return hashlib.sha256(byte_str).hexdigest()[:_CONFIG_ADDRESS_PART_SIZE]
+
+_CONFIG_ADDRESS_PADDING = _config_short_hash(byte_str=b'')
+
+
+def _config_key_to_address(key):
```

Comment:
@jsmitchell I had a quick Slack chat with @peterschwarz regarding that we should consider future story to bring config address, etc.., stuff out to somewhere common.